openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -277,10 +285,17 @@ def _parent_in_hierarchy(self, placeholder, container):
             matrix=True,
             worldSpace=True
         )
+        scene_parent = cmds.listRelatives(
+            placeholder.scene_identifier, parent=True
+        )
```

Comment:
This should take a `fullPath=True` argument. Same goes for the `maya.cmds.listRelatives` calls above to ensure there's never a naming conflict if nodes did not have unique names.

Reply:
Yup thanks